### PR TITLE
Add more unittest and support Qwen3

### PR DESCRIPTION
**Pull request by pan-x-c**
## Description

Add unit tests for
- queue buffer
- explorer
- trainer

Fix bugs
- eval in `explore` mode
- queue buffer writer
- trainer can't stop
- logits 0.0 when setting top-p/k
- runner pool hang

Support Qwen3 series

## Checklist

Please check the following items before code is ready to be reviewed.

- [x]  Code has passed all tests
- [x]  Docstrings have been added/updated in Google Style
- [x]  Documentation has been updated
- [x]  Code is ready for review

### CODE REVIEW

File: tests/common/vllm_test.py [hunk 1]
```diff
@@ -5,15 +5,14 @@
...
-from trinity.common.config import load_config
 from trinity.common.models import create_rollout_models
 from trinity.common.models.model import ModelWrapper
 from trinity.common.models.utils import (
     tokenize_and_mask_messages_default,
     tokenize_and_mask_messages_hf,
 )
 
-config_dir = os.path.join(os.path.dirname(__file__), "tmp", "template_config.yaml")
+from ..tools import get_template_config
```
Comment (chenyushuo) [inline tests/common/vllm_test.py]: Use absolute import

**Review by chenyushuo — CHANGES_REQUESTED** [review]

**Review by yanxi-chen — APPROVED** [review]
lgtm

File: tests/explorer/runner_pool_test.py [hunk 2]
```diff
@@ -31,7 +31,8 @@ def run(self) -> List[Experience]:
         if "timeout" in self.error_type:
             time.sleep(self.seconds)
         elif self.error_type == "exception":
-            raise RuntimeError("Exception occurred")
+            print("rasing exception")
```
Comment (yanxi-chen) [inline tests/explorer/runner_pool_test.py]: typo, rasing --> raising

**Review by chenyushuo — APPROVED** [review]
